openstates/mt/legislators.py

```diff
@@ -112,6 +118,7 @@ def scrape(self, chamber, term):
 
             office['phone'] = deets.get('phone')
             office['fax'] = deets.get('fax')
+            office['email'] = deets.get('email')
             legislator.add_office(**office)
```

Comment:
nope, entry comes from a spreadsheet row and there's no column for email, so I'm going to merge!